src/main/g8/build.sbt

```diff
@@ -1,10 +1,6 @@
 name := "$name;format="lower,hyphen"$"
 version := "0.0.1-SNAPSHOT"
 
-scalaVersion := "2.13.10"
+scalaVersion := "3.3.6"
 
-libraryDependencies += "org.typelevel" %% "cats-core" % "2.9.0"
-
-// scalac options come from the sbt-tpolecat plugin so need to set any here
```

Comment:
Let me know if would like to preserve this comment, or maybe move it directly to the `plugins.sbt`.